snooty.toml

```diff
@@ -30,7 +30,7 @@ toc_landing_pages = ["/quickstart",
 [constants]
 version = "1.9"
 version-previous = "1.8"
-latest-version="1.8.0"
+latest-version="1.9.0"
```

Comment:
feel free to ignore this - bumping the latest c2c version since master points to 1.9